packages/factory_reset_tools/lib/pages/done_page.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:factory_reset_tools/l10n/factory_reset_tools_localizations.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:ubuntu_utils/ubuntu_utils.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru/yaru.dart';
+
+class DonePage extends ConsumerWidget {
+  const DonePage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final lang = FactoryResetToolsLocalizations.of(context);
+    final theme = Theme.of(context);
+
+    return WizardPage(
+      title: YaruWindowTitleBar(title: Text(lang.windowTitle)),
+      content: Row(
+        children: [
+          const Spacer(),
+          Expanded(
+            flex: 7,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                ConstrainedBox(
+                  constraints: const BoxConstraints.tightFor(height: 150),
+                  child: SvgPicture.asset(
+                    'assets/images/mascot_no_background.svg',
+                  ),
+                ),
+                const SizedBox(),
+                Text(
+                  lang.resetMediaReadyTitle,
+                  style: theme.textTheme.headlineSmall,
+                  textAlign: TextAlign.center,
+                ),
+                Text(
+                  lang.resetMediaReadyBody,
+                  textAlign: TextAlign.center,
+                ),
+                SizedBox(
+                  width: 100,
+                  child: FilledButton(
```

Comment:
I'm no designer, but I think an `ElevatedButton` would be more appropriate here

Reply:
It does look a lot better, even though it is not according to the design, but on the other hand I barely followed the design at all because it wasn't ready. Hopefully the designer that reviews this won't kill me. :sweat_smile: